core/src/executor/evaluate/expr.rs

```diff
@@ -68,6 +72,10 @@ pub fn binary_op<'a>(
 }
 
 pub fn unary_op<'a>(op: &UnaryOperator, v: Evaluated<'a>) -> Result<Evaluated<'a>> {
+    if v.is_null() {
+        return Ok(Evaluated::null());
+    }
```

Comment:
this is also the role of `Evaluated::{unary_plus, unary_minus}` to take. like the binary operators, this would be also redundant.

Reply:
Removed in commit [68d9f87](https://github.com/gluesql/gluesql/pull/1577/commits/68d9f872e73b1660e75a024e68eeb79f470ff546)